test/internal/modules/charge-information/controllers/charge-information-workflow.js

```diff
@@ -27,7 +27,9 @@ const createRequest = () => ({
   },
   query: {},
   pre: {
-    chargeInformationWorkflows: [],
+    chargeInformationWorkflows: { data: [{ status: 'to_setup' }, { status: 'to_setup' }, { status: 'to_setup' }], pagination: { totalRows: 1 } },
```

Comment:
Please consider breaking up the stuff here on lines 30,31,32 into multiple lines to make the object more readable.